### ISSUE TITLE
Accessibility: tab test fails when trying to select Generic Work from the Work Type modal 

### ISSUE THREAD

**Issue by eporter23** [samvera/hyrax#6890]
### Descriptive summary

As part of general testing from the QA worksheet with Hyrax 5.0.1, a user cannot successfully tab to the Generic Work selection option, but only can tab to the Monograph option. 

### Steps to reproduce the behavior in User Interface (UI)
See also steps in test A_1.6 from the Hyrax Maintenance working group QA worksheet.

1. Login as a user with deposit priveleges
2. Navigate to the homepage
3. Use the keyboard to tab to Share your Work
4. In the modal window, tab to Generic Work
5. Observe that you cannot tab past the Monograph work type

### Actual behavior (include screenshots if available)

Hyrax 5.0.1 - Sirenia docker image (main)

### Acceptance Criteria/Expected Behavior

- [ ] A user should be able to tab to select any Work Type listed

### Rationale (for feature request only)

Provide the rationale or user story that describes "why" this issue should be addressed. Especially if this is a new feature or significant change to the existing implementation.

Hyrax should support standard accessibility features including tab-based navigation

### Related work

Link to related issues or prior related work here.

**Comment by aahurford** [comment]
I'm seeing that you can tab to the generic work (the first option) but not past it. 

![Image](https://github.com/user-attachments/assets/d6893ab7-726d-4e4c-b3d9-ee8682ff432d)

**Comment by jlhardes** [comment]
Using the Tab key doesn't seem to be the way to work through that list of options when selecting a work type. You use the Tab key to get into the list and then the up and down arrow keys to move down through the options. I can do this now in both dev.nurax and pg.nurax. I will also test locally but I think this actually does pass keyboard accessibility.

**Comment by christhepianist** [comment]
I agree with Julie. The important thing is that it is keyboard-accessible and it is. This is not a problem.
